README.adoc

```diff
@@ -0,0 +1,108 @@
+= Liquibase Runner
+
+--
+[WARNING]
+Older versions of this plugin may not be safe to use. Please review the following warnings before using an older version: +
+* https://jenkins.io/security/advisory/2018-03-26/#SECURITY-519[Plugin allows users to load arbitrary Java code into master JVM] +
+* https://www.jenkins.io/security/advisory/2020-09-23/#SECURITY-2030[Missing permission check allows enumerating credentials IDs] +
+* https://www.jenkins.io/security/advisory/2020-09-23/#SECURITY-1887[XXE vulnerability] +
+* https://www.jenkins.io/security/advisory/2020-09-23/#SECURITY-1885[Stored XSS vulnerability]
+--
+
+[.conf-macro .output-inline]#Provides https://www.liquibase.org/[Liquibase] build steps that evaluate liquibase changesets.#
+
+*Important Upgrade Information* +
+Version 1.2.0 now uses the https://plugins.jenkins.io/credentials[Credentials Plugin] to manage usernames and passwords to connect to the database. If you were using username & password configuration in the previous version of this plugin, be aware that credentials will be automatically created upon plugin update.
+
+[[LiquibaseRunner-Features]]
+== Features
+
+Adds liquibase related operations (update, rollback, and dbdoc generation) as available build steps. See Liquibase documentation at https://www.liquibase.org/[https://www.liquibase.org]. +
+Evaluated changesets and their details appear on the build's summary page.
+
+Uses liquibase version 3.8.9.
+
+[frame="none",grid="none",width="100%",cols="25%,25%,25%,25%",]
+|===
+|image:src/docs/images/dbdoc.png[image,width=300,height=149]
+|image:src/docs/images/changeset-detail.png[image,width=300,height=134]
+|image:src/docs/images/changeset-listing.png[image,width=300,height=138]
+|image:src/docs/images/changeset-summary.png[image,width=300,height=194]
```

Comment:
Renders as 2x2 to make them bigger?  Too small now IMHO

Reply:
I just have made as in the original confluence page.
But changed to 2x2